server/weaverbird/backends/sql_translator/steps/utils/query_transformation.py

```diff
@@ -185,3 +186,10 @@ def complete_fields(query: SQLQuery, columns=None) -> str:
         compiled_query = ', '.join(query_keys)
 
     return compiled_query
+
+
+def handle_zero_division(formula: str) -> str:
+    if '/' not in formula:
```

Comment:
Question: we could div by 0, but for mod it is possible?

Reply:
Not it isn't, I'll update that.